src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -223,7 +232,11 @@ export const PatientInfoTable = ({
   };
 
   const handleCloseDetails = () => {
-    history.push(`/events/${eventId}/ccps/${ccpId}`);
+    if (patientId) {
+      history.push(`/events/${eventId}/ccps/${ccpId}`);
```

Comment:
@jesskeepswimming ?

Reply:
yup--since we are changing the dialog behavior to only redirect if opened from a snackbar link, there is no need to his.tory.push if there is no patientId passed in aka not from the table